service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/RespondentSolicitorContentProvider.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
+import uk.gov.hmcts.reform.fpl.model.notify.submittedcase.RespondentSolicitorTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.SharedNotifyContentProvider;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+
+public class RespondentSolicitorContentProvider extends SharedNotifyContentProvider {
+
+    public RespondentSolicitorTemplate buildNotifyRespondentSolicitorTemplate(CaseData caseData,
+                                                                              RespondentSolicitor representative) {
+
+        return RespondentSolicitorTemplate.builder()
+            .representativeName(representative.getLastName())
```

Comment:
in #2481 this looks like:

`.representativeName(representative.getFirstName() + " " + representative.getLastName())`

what is the correct format, for consistency? (equivalent comment left on that PR)

Reply:
https://github.com/hmcts/fpl-ccd-configuration/pull/2481/files#r610481712

Reply:
Checked with someone and it should be the full name

Reply:
Should we have something like

`@JsonIgnore
    public String getFullName() {
        return String.format("%s %s", defaultString(firstName), defaultString(lastName)).trim();
}`

unless you are fixing the issue where name can be blank via mid event Soorya?